gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -444,6 +469,30 @@ gst_tensor_converter_set_property (GObject * object, guint prop_id,
       self->silent = g_value_get_boolean (value);
       silent_debug ("Set silent = %d", self->silent);
       break;
+    case PROP_MODE:
+    {
+      const gchar *param = g_value_get_string (value);
+      gchar **strv = g_strsplit_set (param, ":", -1);
+      const converter_custom_cb_s *ptr = NULL;
+
+      self->mode = g_strdup (strv[0]);
```

Comment:
check length of strv before using strv[n]

Reply:
Updated.